assets/app/view/game/buy_sell_shares.rb

```diff
@@ -30,7 +30,8 @@ def render
 
         children << h(SellShares, player: @current_entity, corporation: @corporation)
 
-        h(:div, children.compact)
+        children = children.compact
+        h(:div, children) if children.any?
```

Comment:
this is not really well defined behavior as it implicitly returns nil i believe, you shouldn't dot his pattern,  always return a div

Reply:
or if you do need to return the nil explicitly and fix all references of it to handle it